isolate_test.go

```diff
@@ -39,6 +39,9 @@ func TestIsolateTermination(t *testing.T) {
 	if e := <-err; e == nil || !strings.HasPrefix(e.Error(), "ExecutionTerminated") {
```

Comment:
By the way, we could move ctx.RunScript out of the unnecessary goroutine it is called in just above this line

```suggestion
	_, e := ctx.RunScript(`while (true) { }`, "forever.js")
	if e == nil || !strings.HasPrefix(e.Error(), "ExecutionTerminated") {
```